ui/src/overridden/backoffice/Importer/ImportedDocuments.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Accordion, Button, Header, Icon, Message } from 'semantic-ui-react';
+import _isEmpty from 'lodash/isEmpty';
+import { http } from '@inveniosoftware/react-invenio-app-ils';
+import _get from 'lodash/get';
+import { ReportDetails } from './ReportDetails';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@inveniosoftware/react-invenio-app-ils';
+import { DocumentIcon } from '@inveniosoftware/react-invenio-app-ils';
+
+const importerURL = '/importer/check';
+
+export class ImportedDocuments extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeIndex: -1,
+      importCompleted: false,
+      data: null,
+      intervalId: null,
+      isLoading: true,
+    };
+  }
+
+  componentDidMount() {
+    //TODO: Define how much we wait until the next call (currently 30 sec)
+    const idVar = setInterval(() => this.checkForData(idVar), 30000);
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ intervalId: idVar });
+    this.checkForData(idVar);
+    window.addEventListener('beforeunload', this.onCloseWarning);
+  }
+
+  componentWillUnmount = () => {
+    const { intervalId } = this.state;
+    window.removeEventListener('beforeunload', this.onCloseWarning);
+    clearInterval(intervalId);
+  };
+
+  onCloseWarning = e => {
+    const { isLoading } = this.state;
+    if (isLoading) {
+      var confirmationMessage =
+        'Importing in progress. Are you sure you want to leave?';
+
+      (e || window.event).returnValue = confirmationMessage; //Gecko + IE
+      return confirmationMessage; //Webkit, Safari, Chrome
+    }
+  };
+
+  checkForData = async idVar => {
+    const { importCompleted } = this.state;
+    const { taskId } = this.props;
+
+    if (!importCompleted) {
+      // eslint-disable-next-line no-useless-concat
+      const response = await http.get(`${importerURL}` + '/' + `${taskId}`);
```

Comment:
I would have maybe created a nice endpoint in the `api.py` as for the others, instead of having it inside a component.